class.jetpack.php

```diff
@@ -6933,21 +6933,24 @@ public static function build_raw_urls( $url ) {
 	public static function dns_prefetch( $new_urls = null ) {
 		static $prefetch_urls = array();
 		if ( empty( $new_urls ) && ! empty( $prefetch_urls ) ) {
-			echo "\r\n";
-			foreach ( $prefetch_urls as $this_prefetch_url ) {
-				printf( "<link rel='dns-prefetch' href='%s'/>\r\n", esc_attr( $this_prefetch_url ) );
+			if( 'dns-prefetch' == $relation_type ) {
```

Comment:
Where does `$relation_type` come from?

Reply:
I've pushed another commit up, with this value statically declared now. However I don't feel like it's the best approach - it's potentially a value that you could pass back as another parameter (alongside the URLs) for differing types of `<link>`?
Implementing this is a _little_ outisde of my current comfort zone, though I'll keep looking/trying!

Reply:
So, I have now statically declared that in the latest commit, however I don't feel like that's the best approach as it's inflexible. Say in the future you want to pass `preconnect` or another `<link>` type, we could extend the `Jetpack::dns_prefetch` to pass that variable back?
Bit outside of my current comfort zone, but I'll give it a whirl :)